src/analyses/baseInvariant.ml

```diff
@@ -691,7 +697,30 @@ struct
               | TFloat (fk, _) -> `Float (FD.of_int fk c)
               | _ -> `Int c
             in
-            update_lval c x c' ID.pretty
+            let st = update_lval c x c' ID.pretty in
+            (* handle special calls *)
+            begin match t with
+            | TInt (ik, _) ->
+              begin match x with
+                | ((Var v), offs) -> 
+                  if M.tracing then M.trace "invSpecial" "qry Result: %a\n" Queries.ML.pretty (ctx.ask (Queries.TmpSpecial (v, TmpSpecial.Spec.resolve offs))); 
+                  let tv = not (ID.leq c (ID.of_bool ik false)) in
```

Comment:
Is this correct? Consider, e.g. `c = [0,1]`?
$tv = \neg ([0,1] \not\sqsubseteq [0,0]) \implies tv = \textsf{true}$

Reply:
This was a mistake on my side. Now it is handled correctly with `ID.to_bool`